assets/img/.gitkeep

```diff
@@ -0,0 +1 @@
+
```

Comment:
I think you can get rid of this file as you already have `assets/img/header` folder with some icons there.